### DISCUSSION TITLE
Why I'm shown activity of users I don't follow?

### DISCUSSION THREAD

**Discussion by gvolpe** [community/community#13152] [Other Feature Feedback, Questions, & Ideas] [answered]
I would only like to see activity of people I follow, and it would be awesome if you finally add support for following organizations. I think that's the whole point of the "follow" button, no?

Here's an example where I see activity of `scala-steward`, which is an organization I don't/can't follow.

![boo](https://i.imgur.com/Xh6bWHm.png)

The rest of the feed is pure noise IMO.

**Comment by bdresser** [comment]
Hey @gvolpe, thanks for the feedback. 

You will only see activity from people you follow and repos you watch in the "following" tab — this activity has not changed. 

Keep an eye out for following organizations... this will be landing shortly 👀

**Reply by gvolpe** [reply]
Hmm that doesn't explain why I'm being shown activity of the organization https://github.com/scala-steward ... Is it because I'm watching the two repos it forked? I'm only watching them with notification settings set to "Participating and @mentions". It would be great if these could be excluded from the new feed.

FWIW I don't see this activity in the current feed.

**Reply by MattIPv4** [reply]
![image](https://user-images.githubusercontent.com/12371363/159347069-6ed4c861-895b-4678-a20f-aa00089868b8.png)

![image](https://user-images.githubusercontent.com/12371363/159347444-57e51606-2036-46ac-83e0-1623ac0921b9.png)


I do not follow these users, nor do I follow the repos. My for you feed is full of items like this from users and repos I don't recognise.

**Reply by bdresser** [reply]
@MattIPv4 yup, this is expected! Our goal here is to help folks encounter projects and activity they might be able to learn from. Specifically, you'll see events from people you've collaborated with and repositories you've contributed to in the past. 

If you're not interested in this activity, head back to your "Following" feed, which you have complete and explicit control over. Will link to docs shortly with a bit more detail.

**Reply by bdresser** [reply]
@gvolpe following an organization has now shipped :)

**Reply by MattIPv4** [reply]
Ah interesting -- I don't recognise these users or repos at all as things I've interacted with in the past. Given GitHub's developer audience, might be beneficial to be more transparent in the UI about why items are showing up (e.g. "This user contributed to repo xyz which you maintain") or smth.

**Reply by JasonEtco** [reply]
@gvolpe I think you're seeing the same behavior as @MattIPv4, where you're seeing activity for a user (`@scala-steward` is a bot user, not an org) that you don't explicitly follow but is "part of your overall network". Agreed that more UI clarity would help a ton.

**Reply by jamieshark** [reply]
@MattIPv4 - we have current plans to show the reason why you are seeing a certain item in your "For You" feed, so stay tuned. Thanks for letting us know this would be useful to you.

**Reply by gvolpe** [reply]
> If you're not interested in this activity, head back to your "Following" feed, which you have complete and explicit control over.

@bdresser does this mean both feeds are going to be kept alongside? I wouldn't mind that. I thought the new feed was a replacement of the current one, in which case, a lot of users would not be happy about that (including me). 

> following an organization has now shipped :)

That's great news! Thank you ❤️

**Reply by rgwood** [reply]
@bdresser I am really, really concerned that the Following feed is going to be removed in favour of this algorithmic feed. People are going to assume the worst if GitHub does not offer clarity on this point.

**Comment by askpt** [comment]
I like visually the new design of the new feed, but it's confusing why some items appear in the feed itself. Perhaps if we press the 3 dots in the top right corner and have an explanation why we are seeing that card would be helpful.

**Reply by jamieshark** [reply]
@askpt - we have this work currently planned and will be rolled out soon. 🙂

**Comment by james-s-w-clark** [comment]
My "For you (Beta)" feed is dominated by dependency bots, with items like:
![image](https://user-images.githubusercontent.com/12559216/182148841-5ff1addf-3694-4602-b6ad-946de5ee2a02.png).

A few times (over a couple days) I tried to go to the [bot user's page](https://github.com/scala-steward) to "unfollow" it, and I got the Github Unicorn page. It's OK now.

I'm not following the bot - and I don't want to see posts from it in the feed. 
I would be happy seeing projects and activity I might be able to learn from or help with - but this kind of noise subtracts from, rather than adds to that experience.

**Comment by jfullstackdev** [comment, marked as answer]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by james-s-w-clark** [reply]
Thanks for the update.
This works best for me (it hides Dependabot/Scala Steward type spam):
![image](https://user-images.githubusercontent.com/12559216/182222540-84bd0660-f1b8-46bd-a69c-00532347720b.png)

**Reply by gvolpe** [reply]
@jdevstatic thanks for your reply, I think the idea is great. 

However, after giving it a try, I must say it's got a few areas that need improvement. 

---

Here's an example where I'm being shown unwanted content.

1. I applied some filters

![filters](https://i.imgur.com/8GGWUrk.png)

2. I see something about [Z3 Theorem Prover](https://github.com/Z3Prover), an organization I do not follow, and that I've never even heard of.

![feed](https://i.imgur.com/kXaWiZA.png)

3. Following the link to the organization, I can corroborate I do not follow it.

![org](https://i.imgur.com/8dO8HXg.png)

So the questions is, **why is it on my feed**?

---

Another big improvement that has been suggested by many users would be to indicate WHY a content is being displayed in a users' feed. Because right now, it is impossible to understand why I see unwanted content on my feed.

Hope that makes sense, thanks!

**Reply by jfullstackdev** [reply]
yup, there will still be new updates for sure, there were several requests regarding those, but the filter is already the first major update I think so
